### PR TITLE
Project: MediaWiki

### PR DESCRIPTION
**Pull request by AdrienLemaire**
PR 6/15 for #29

### CODE REVIEW

File: src/mediawiki.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+contributionGuidelinesUrl: "https://www.mediawiki.org/wiki/How_to_contribute"
+logoUrl: "https://www.mediawiki.org/static/images/project-logos/\
+          mediawikiwiki.png"
+
+ratingComment: This project is a model of OSS development
+frameworksUsed:
+  - Swing
+  - Lua
+  - LESS
```
Comment (OscarAblinger) [inline src/mediawiki.yaml]: I assume `Lua` and `LESS` were meant for `programmingLanguages` not `frameworksUsed`? ^^

**Review by OscarAblinger — CHANGES_REQUESTED** [review]

File: src/mediawiki.yaml (hunk 1 above)
Comment (AdrienLemaire) [reply src/mediawiki.yaml]: hmm... this warrants a discussion. Lua is only using in templates, and LESS is a CSS framework, so I wouldn't put them in the same category as other programming languages.
I'll just remove them for now :)

Comment (OscarAblinger) [reply src/mediawiki.yaml]: I don't know enough about Lua to say much about it. I just looked it up and its website described it as a programming language 🤷‍♂ If you think it's more of a framework then leave it in there as that.

LESS, however, I'd definitely not define as a framework. In the end it only generates a file that is included into the HTML later on. It doesn't provide any generic functionality that is extended by its users.

**Comment by OscarAblinger** [comment]
I think we should add Lua & Less somewhere (either frameworks or programming languages, since people might want to look for it 🤔

**Review by OscarAblinger — APPROVED** [review]

**Pull request merged**